dojson/overdo.py

```diff
@@ -52,6 +65,7 @@ def do(self, blob):
         output = {}
 
         if self.index is None:
+            self._collect_entry_points()
```

Comment:
I did it like this because I thought it would be nice to have the ability to decide when to collect and when to build if one wants to do it manually for whatever reason, that's why I made the `collect` method "public" at the beginning.
Now that the method was moved to "private" world, I'm not sure if this makes sense any more although some part of me still likes the idea of small independent steps